packages/compass-indexes/src/components/indexes-table/indexes-table.tsx

```diff
@@ -134,17 +135,17 @@ export const IndexesTable: React.FunctionComponent<IndexesTableProps> = ({
               properties={index.properties}
             />
           </Cell>
-          {/* Delete column is conditional */}
-          {index.name !== '_id_' && canDeleteIndex && (
-            <Cell data-testid="index-drop-field" className={cellStyles}>
-              <div className={cx(deleteFieldStyles, 'delete-cell')}>
-                <DropField
-                  name={index.name}
-                  onDelete={() => onDeleteIndex(index.name)}
-                />
+          <Cell data-testid="index-actions-field" className={cellStyles}>
```

Comment:
@alenakhineika @mabaasit does this cause too many conflicts?

Reply:
i dont think it will cause any conflicts. 

But this change (to show cell always), will also be rendered even if user can not delete an index (`canDeleteIndex  = false`), which i think will break the table layout.

Reply:
I see how would the layout break? Like the header won't be visible?

Reply:
yes exactly. if we split the check, to show cell based on `canDeleteIndex` and then for `_id` just hide the cell contents

Reply:
This PR can wait, is only required for the rebranding and is not changing anything at the moment, I can rebase from Alena's when it lands

Reply:
Ok fixed the column issue!

Reply:
Can we always have the delete header and the delete table cell? Maybe to now have then only in case whencanDeleteIndex is not set?

Reply:
but now when i read your comment again this is exactly what you meant)

Reply:
Oh, that makes sense cause if the index creation fails then we need that column, I think we can always keep that yeah. I'll wait for the other PR and rebase then!